src/libraries/Native/Unix/System.Globalization.Native/pal_icushim.h

```diff
@@ -8,8 +8,9 @@
 
 #pragma once
 
+// if defined we're building the qcalls entrypoints and we don't need to include icu or config headers
```

Comment:
I thought that you have done the renaming to `pal_locale_internal` so that this is no longer necessary?

Reply:
`pal_icushim.h` is still the one that brings the `ICUHeaders`. Which we use in all our headers and implementations. However when doing the entrypoints build I define UChar at the bottom of this file. If you want me to that, I could move the ICU header inclusions file to a different header file and do the right things in order to not include config.h nor these headers when doing the entrypoints build.

Reply:
I will leave it up to you.

Reply:
Thanks. I will see how it looks with moving the icu headers imports to a new file. Then I'll decide, anyway, I would still like to use `#ifdef __LIB_NATIVE_ENTRYPOINTS` to define `UChar` to make sure that that type is only defined in our side when building the entrypoints.

Reply:
At the end I decided to move it to `pal_icushim_internal.h` and get rid of `__LIB_NATIVE_ENTRYPOINTS` so that entrypoints.c and globalization are de-coupled. Thanks